Add governance for sig-agentic-ai

haskalpa opened the pull request:
Establishing a new CNOE sig.

haskalpa commented on sigs/agentic-ai/charter.md:
```diff
@@ -0,0 +1,45 @@
+- [cncf #cnoe-sig-agentic-ai](https://cloud-native.slack.com/archives/C08N0AKR52S)
+
+
+## Scope
```
This is addressed @greghaynes , will add sub-projects as needed

greghaynes commented:
Sorry one more thing I noticed - can you also add this SIG to the readme here https://github.com/cnoe-io/governance

greghaynes reviewed (approved):
LGTM, would be great to add to the main readme though.

Per governance I think we need majority of member orgs to approve to add this SIG.

haskalpa commented:
> LGTM, would be great to add to the main readme though.
> 
> Per governance I think we need majority of member orgs to approve to add this SIG.

thanks @greghaynes , added and fixed idp-builder one to point to README as well

nabuskey reviewed (approved):
LGTM

csantanapr commented:
LGTM thank you CISCO for selecting CNOE to innovate in this new space of AI agents
